.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,64 @@
+<!--
+
+Thank you for contributing to the Firebase community!
+
+Have a usage question?
+=======================
+We get lots of those and we love helping you, but GitHub is not the best place for them and they
+will be closed. Here are some resources to get help:
+
+- Start with the quickstart: https://firebase.google.com/docs/functions/write-firebase-functions
+- Go through the guide: https://firebase.google.com/docs/functions/
+- Read the full API reference: https://firebase.google.com/docs/reference/functions/
+- Browse some examples: https://github.com/firebase/functions-samples
+
+If the official documentation doesn't help, try asking through our official support channels:
+
+https://firebase.google.com/support/
+
+*Please avoid double posting across multiple channels!*
+
+Think you found a bug?
+=======================
+Yeah, we're definitely not perfect! Please use the bug report template below and include a minimal
+repro when opening the issue.
+
+
+Have a feature request?
+========================
+Great, we love hearing how we can improve our products! We have a public issues tracker for you to
+post your suggestions to: https://issuetracker.google.com/issues?q=componentid:184396
+
+-->
+
+
+### Version info
+
+<!-- What versions of the following libraries are you using? Note that your issue may already
+be fixed in the latest versions. -->
+
+**firebase-functions:**
+
+**firebase-tools:**
+
+**firebase-admin:**
+
+### Test case
+
+<!-- Provide a minimal, complete, and verifiable example (http://stackoverflow.com/help/mcve) -->
+
+
+### Steps to reproduce
+
+<!-- Provide the steps needed to reproduce the issue given the above test case. -->
+
+
+### Expected behavior
+
+<!-- What is the expected behavior? -->
+
+
+### Actual behavior
+
+<!-- Please copy and paste any error logs from https://console.firebase.google.com/project/_/functions/logs.
```

Comment:
It might make sense to add a question to the template which says "Did you run into this problem while deploying your Function or while your Function was executing?"